bricksrc/rules.ttl

```diff
@@ -106,7 +106,7 @@ CONSTRUCT {
 bsh:DeprecationRule
     a   sh:NodeShape ;
     sh:property [
-        sh:path brick:deprecation ;
+        sh:path (rdf:type brick:deprecation) ;
```

Comment:
Please add more description to this in the PR or as a comment in the code. It seems to be changed from the original deprecation design.

Reply:
It just fixes a bug where deprecations warnings would fire if you had the deprecation definitions in the graph --- not when you actually instantiated a deprecated concept